src/hb-ot-color-colr-table.hh

```diff
@@ -150,9 +611,11 @@ struct COLR
   bool sanitize (hb_sanitize_context_t *c) const
   {
     TRACE_SANITIZE (this);
```

Comment:
I believe this still needs a check_struct() to make sure it's safe to access the offset fields. Will need to check beyond the min size if it's a v1 table.

Reply:
Either the check_struct, or individually checking each item. The latter is what we do in GDEF / GSUB / GPOS. Should also check version == 0 for the first few fields, then version == 1 for the newer ones.